modules/contact/src/ContactMaster.C

```diff
@@ -417,6 +417,9 @@ contactFormulation(const std::string & the_name)
   else if ("augmented_lagrange" == name)
     formulation = CF_AUGMENTED_LAGRANGE;
 
+  else if ("tangential_penalty" == name)
```

Comment:
> I will double-check this now.

Yeah, they provide `operator int() const { return _current_id; }` and this gets called when you do `switch(foo)` where `foo` is a `MooseEnum`.